src/content/docs/journey-migration/migrating-dashboards.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: "Create alerts and dashboards"
```

Comment:
Same issue as others, I think this is riskily generic.

Reply:
Fixed!